test/model/model_test.dart

```diff
@@ -0,0 +1,35 @@
+import 'package:test/test.dart';
+
+import 'package:casbin/src/enforcer.dart';
+
+import '../utils/test_utils.dart';
+
+void main() {
+  final modelPath = 'examples\\basic_model.conf';
+  final policyFile = 'examples\\basic_policy.csv';
```

Comment:
@MrUnfunny I had to manually change the file paths to the linux convention (forward slash) to run the tests.

Reply:
We should change this to something that makes it universally same for all platforms @MrUnfunny

Reply:
Hi @santhoshivan23, were you on Linux? For me, both '\\' and '/' works fine. I'll change it to '/'. It works on all systems. Right?

Reply:
yep!